sources/packages/api/src-sql/AuthCodeTables/Drop-institution-user-type.sql

```diff
@@ -0,0 +1,2 @@
+-- DROP Table
+DROP TABLE IF EXISTS "institution_user_type_role";
```

Comment:
I believe that there is an "s" missing in the name. The correct name of the table to be dropped is "institution_user_type_roles".